kafka_consumer_freshness_tracker/README.md

```diff
@@ -47,7 +47,16 @@ bazel-bin/kafka_consumer_freshness_tracker/src/main/java/com/tesla/data/consumer
 
 Bundled as a jar you can run the Tracker with a configuration file with the `--conf <path to conf>` flag. For local 
 testing, you can also just add the `--once` flag to just run the Tracker once, and then dump the metrics to the console, 
-before stopping.
+before stopping. 
+
+## Validation flags
```

Comment:
Reading the latest version....Its odd that all the validations are described as "failures" and then at the very end, is it mentioned that the application only fails  when `--strict`. I think it would be nicer to describe the validations that are done (e.g. "it checks a and b and c) and then describe the choices you have around that handling the found issues (e.g. by default, these validation errors are just logged, but `--strict` will fail the application to fail as well)